### ISSUE TITLE
How to render background?

### ISSUE THREAD

**Issue by ScienceLi1125** [Jumpat/SegAnyGAussians#37]
Hi! I greatly appreciate your innovative work.

Under your instruction, I can render the target scene by `python render.py -m output/XXX --precomputed_mask segmentation_res/final_mask.pt --target scene --segment`. But if I want to render the **background** instead of the target, that is somewhere not selected by target Gaussians, how can I achieve it? 

I tried to replace 0 with 1 and 1 with 0 in `mask` in `gaussian_renderer/__iniy__.py`, but it didn't work. (`mask = torch.logical_not(mask.bool()).float()`)

Thanks for your help.

**Comment by Jumpat** [comment]
Thanks you.👍

Check this [line](https://github.com/Jumpat/SegAnyGAussians/blob/5142b1ab1e098b2fef191feeee8113b2fc2be137/scene/gaussian_model.py#L335C9-L335C16) you can find what you need.

**Comment by ScienceLi1125** [comment]
It works! Thanks a lot.
